lib/main.dart

```diff
@@ -31,47 +31,19 @@ class MainPage extends StatefulWidget {
 class _MainPageState extends State<MainPage> {
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: Text(widget.title),
-      ),
-      body: Background(
-        child: Center(
-          child: CircularList(
-            centerWidget: TeamMember(name: 'Scrum master'),
-            children: List.generate(8, (index) {
-              return TeamMember(
-                name: 'Person $index',
-              );
-            }),
-          ),
-        ),
-      ),
-      floatingActionButton: OpenContainer(
-        transitionType: ContainerTransitionType.fade,
-        openBuilder: (BuildContext context, VoidCallback _) {
-          return AddMember();
-        },
-        closedElevation: 6.0,
-        closedShape: const RoundedRectangleBorder(
-          borderRadius: BorderRadius.all(
-            Radius.circular(_fabDimension / 2),
-          ),
-        ),
-        transitionDuration: Duration(seconds: 1),
-        closedColor: Theme.of(context).colorScheme.secondary,
-        closedBuilder: (BuildContext context, VoidCallback openContainer) {
-          return SizedBox(
-            height: _fabDimension,
-            width: _fabDimension,
-            child: Center(
-              child: Icon(
-                Icons.add,
-                color: Theme.of(context).colorScheme.onSecondary,
-              ),
-            ),
+    return BlocProvider(
+      create: ((BuildContext context) {
+        return TeamMembersBloc()
+          ..add(
+            TeamMembersEventLoadData(),
```

Comment:
Not sure who should trigger this.. I guess later, when we try to persist data, then we would try to load the data on app start and once loaded we would trigger this..

Reply:
in the past i added a ..start() event during creation which started loading the data and when finished triggered a loaded event